OpenRA.Mods.Common/Warheads/SpawnActorWarhead.cs

```diff
@@ -0,0 +1,221 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.GameRules;
+using OpenRA.Mods.Common.Activities;
+using OpenRA.Mods.Common.Effects;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Warheads
+{
+	[Desc("Spawns actors upon detonation.")]
+	public class SpawnActorWarhead : Warhead, IRulesetLoaded<WeaponInfo>
+	{
+		[FieldLoader.Require]
+		[Desc("Actors to spawn.")]
+		public readonly string[] Actors;
```

Comment:
Type of actors? Or actor (number of) instances?

Reply:
Both. You list the actors you want to spawn